pytorch_pfn_extras/utils/checkpoint.py

```diff
@@ -16,13 +17,18 @@ class _CheckpointFunction(torch.utils.checkpoint.CheckpointFunction):
     persistent parameters.
     """
     @staticmethod
-    def forward(ctx, run_function, preserve_rng_state, *args):
+    def forward(  # type: ignore[override]
+            ctx: Any,
+            run_function: torch.nn.Module,
```

Comment:
This may cause a problem once type annotation is added `torch.utils.checkpoint.CheckpointFunction`, which expects a callable.

```suggestion
            run_function: Any,
```